### PR TITLE
The Twitter Package

### PR DESCRIPTION
**Pull request by diananeculai**
This package is built over Twitter REST API v1.1

Solves issue #83

### CODE REVIEW

File: src/Joomla/Twitter/Block.php [hunk 1]
```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Part of the Joomla Framework Linkedin Package
```
Comment (realityking) [inline src/Joomla/Twitter/Block.php]: Twitter Package (same issue in other files too)

Comment (diananeculai) [inline src/Joomla/Twitter/Block.php]: Fixed.

**Comment by diananeculai** [comment]
Are there any changes you're thinking of before getting this into the framework? It has been sitting here for some time now.

**Comment by mbabker** [comment]
Personally, I don't think anything needs to change.  I've left a note with the others asking for review, I'm a bit hesitant to just merge it since my name's in the commit history here.  Activity here's been on a lull the last few weeks, life being what it is and all, but I think we can get this merged in the next week.

**Pull request merged**